src/client/components/FavouriteList/ProductImage.js

```diff
@@ -8,8 +8,8 @@ function ProductImage({ product }) {
       <div>
         <img
           className="product_image"
-          src={product.image.src}
-          alt={product.image.alt}
+          src={product.image}
```

Comment:
This is not showing the pictures because the column name is `picture`  not `image`